growth/too/tasks/gattini_client.py

```diff
@@ -0,0 +1,62 @@
+from astropy import time
+import astropy.units as u
+from celery.task import PeriodicTask
+from celery.utils.log import get_task_logger
+
+from ..flask import app
+from . import celery
+from .. import models
+
+import psycopg2
+import psycopg2.extras
+
+log = get_task_logger(__name__)
+
+__all__ = ('gattini_obs')
+
+
+def get_obs(start_time, end_time):
+
+    conn = psycopg2.connect(host=app.config["GATTINI_HOST"],
```

Comment:
Please put this in the SQLAlchemy database binds so that the connection is recycled across HTTP requests. You can still manually issue the SQL query, though.